Kudu.Services.Web/Content/Scripts/ProcessExplorer.js

```diff
@@ -863,6 +870,14 @@ function overrideRightClickMenu() {
                     $("li")[0].click();
                     $("li").blur();
                     break;
+                case "crashdump":
+                    $("#proc-loading").show();
+                    process.enableCrashDump().done(function () {
+                        return processExplorerSetupAsync();
+                    }).fail(function () {
+                        return processExplorerSetupAsync();
```

Comment:
No, they'll only see `procdump.exe` not being launched after the processes refresh. I'll look into displaying the error. The code in `ProcessExplorer.js` is pretty old and was hoping I won't have to mess with it too much :)